etl/work_orders_to_socrata.py

```diff
@@ -95,6 +96,25 @@ def transform_datetime_columns(data):
     return data
 
 
+def cleanup_work_order_urls(data):
+    """
+    Replaces spaces with the proper URL encoding from work order URLs.
+    Socrata will silently reject URLs with spaces.
+    ----------
+    data: list of dicts of the data fetched from Maximo.
+
+    Returns
+    -------
+    data: list of dicts with the urls cleaned up.
+
+    """
+    for row in data:
+        # Only a few work orders have spaces in them.
+        if " " in row["WONUM"]:
+            row["WO_LINK"] = row["WO_LINK"].replace(" ", "%20")
+
+    return data
```

Comment:
i dunno if you have concerns about other characters sneaking in, but [`urllib.parse`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus) could help you future proof it

Reply:
I don't have many concerns. It's only 75 records with spaces and it will only affect this `WO_LINK` field being blank. Socrata has no problem storing other special characters in the URL for some reason.